tools/run_session

```diff
@@ -75,8 +75,7 @@ start_av_recording() {
             # We need to extract the alsa output monitor.
             export ALSA_OUTPUT_MONITOR=`pacmd list-sources | \
             awk '/name:/ && /monitor/ {print $2 }'|sed 's/[<,>]//g'`
-            # Calling pacat command.
-            pacat --record --file-format=wav -d ${ALSA_OUTPUT_MONITOR} \
+            # Calling pacat command.            pacat --record --file-format=wav -d ${ALSA_OUTPUT_MONITOR} \
```

Comment:
Looks like the pacat command got accidentally joined with the comment line above it.

Reply:
Fixed.